GitOut/Features/Git/Log/GitLogPage.xaml

```diff
@@ -202,6 +202,77 @@
                             <Path Fill="{DynamicResource MaterialForegroundBase}" Data="{StaticResource ContentCopy}"/>
                         </Button>
                         <Border Style="{StaticResource MaterialVerticalDivider}"/>
+                        <ToggleButton x:Name="Remotes" Style="{StaticResource MaterialIconToggleButtonStyle}" ToolTip="Remotes">
+                            <Path Fill="{DynamicResource MaterialForegroundBase}" Data="{StaticResource DownloadNetworkOutline}"/>
+                        </ToggleButton>
+                        <Popup
+                            StaysOpen="False"
+                            AllowsTransparency="True"
+                            IsOpen="{Binding IsChecked, ElementName=Remotes}"
+                            PopupAnimation="Fade"
+                            Placement="Bottom"
+                        >
+                            <Grid>
+                                <Border Style="{StaticResource CardPanelStyle}" MinWidth="200" Margin="16 8" Padding="0">
+                                    <Grid>
+                                        <Grid.RowDefinitions>
+                                            <RowDefinition Height="Auto" />
+                                            <RowDefinition Height="Auto" />
+                                            <RowDefinition Height="*" />
+                                        </Grid.RowDefinitions>
+                                        <Grid Grid.Row="0" Margin="8 4">
+                                            <Grid.ColumnDefinitions>
+                                                <ColumnDefinition Width="*" />
+                                                <ColumnDefinition Width="Auto" />
+                                            </Grid.ColumnDefinitions>
+                                            <StackPanel Grid.Column="1">
+                                                <Button
+                                                    Style="{StaticResource MaterialButtonStyle}"
+                                                    Command="{Binding FetchRemotesCommand}"
+                                                >
+                                                    FETCH
+                                                </Button>
+                                            </StackPanel>
+                                        </Grid>
```

Comment:
unless we're expecting more content to be added here, then horizontalalignment can be used to right justify
```suggestion
                                        <StackPanel 
                                            Grid.Row="0"
                                            Margin="8 4"
                                            HorizontalAlignment="Right"
                                        >
                                            <Button
                                                Style="{StaticResource MaterialButtonStyle}"
                                                Command="{Binding FetchRemotesCommand}"
                                            >
                                                FETCH
                                            </Button>
                                        </StackPanel>

```

Reply:
I was thinking of adding options for autofetching and stuff, but we can do that when we get there.